### PR TITLE
Alignments update

### LINKED ISSUE #316: GeoSPARQL to DCTERMS mapping
In the current GeoSPARQL 1.1 draft ([§E.9](https://opengeospatial.github.io/ogc-geosparql/geosparql11/spec.html#_e_9_dcmi_metadata_terms_dcterms)), there is a discussion on how to map [`dcterms:coverage`](https://www.dublincore.org/specifications/dublin-core/dcmi-terms/#http://purl.org/dc/terms/coverage).

I would suggest replacing it with its more relevant subproperty, namely, [`dcterms:spatial`](https://www.dublincore.org/specifications/dublin-core/dcmi-terms/#http://purl.org/dc/terms/spatial), and revise the first sentence as follows:

> The range of `dcterms:spatial` includes a `dcterms:Location`, so it is a property for indicating a `geo:Feature`, not a `geo:Geometry` and for which GeoSPARQL has no equivalent.

About the sentence:

> Often, `dcterms:coverage` is used to indicate a spatial extent such as a bounding box. 

This is not completely correct. What is used for this purpose is usually [`dc:coverage`, i.e., the original term from the Dublin Core Metadata Element Set](https://www.dublincore.org/specifications/dublin-core/dcmi-terms/#http://purl.org/dc/elements/1.1/coverage), whose value is a literal. Instead, `dcterms:coverage`, as well as its subproperty `dcterms:spatial`, are meant to be used as object properties. 

An example of its use is included in the _Dublin Core Guide on Publishing Metadata_ - quoting from [the relevant section](https://www.dublincore.org/resources/userguide/publishing_metadata/#dcterms:spatial):

> dcterms:spatial has a range of the class dcterms:Location. All values used with dcterms:spatial have to be instances of this class. Therefore the property must only be used with non-literal values.
>
> ````turtle
> ex:myData dcterms:title "Analysis of rocks collected in Perth" ;
>           dcterms:spatial _:thisPlace .
> _:thisPlace ex:east "115.85717" ;
> ex:north "-31.95301" ;
> ex:name "Perth, W. A." .
> ````

### LINKED ISSUE #355: geosparql 1.1 BFO mapping of geo:Geometry yields unsatisfiable classes
geo:Geometry subClassOf: 'spatiotemporal region'
geo:Geometry equivalentClasses: 'spatial region'

together, make 'spatial region' subclassOf 'spatiotemporal region'. But spatial region and spatiotemporal region are continuants and occurrants respectively, and those are disjoint.

I wasn't sure how it would be concluded that:
>Note that this mapping is also inherited from geo:Geometry being an equivalent class of obo:BFO_0000006

The skos:note on Geo:Geometry says
>Geo:Geometry can be used as a *representation* of the shape, extent or location of a Feature, or can exist as a self-contained entity.  [my emphasis]

with the definition:

>A coherent set of direct positions in space. The positions are held within a Spatial Reference System (SRS)."

I don't think these are consistent with each other, as least insofar as BFO is concerned. Shapes and extents would be BFO qualities, and locations are independent continuants, which are disjoints. To my reading, a geo:Geometry would be an [information content entity](https://www.ebi.ac.uk/ols/ontologies/genepio/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FIAO_0000030). Information entities are the sorts of things that represent, or are about, other things. There's nothing problematic about different instances of a class of information entities being about different sorts of things.

As an aside, speaking to the wording of the definition, what would an *incoherent* set of positions in space be?

Screen Shot 2022-06-22 at 9 16 40 AM
.

### PR DESCRIPTION
**Pull request by nicholascar**
Updates to mappings based on feedback from mapped-to specifications' experts.

* DCTERMS: https://github.com/opengeospatial/ogc-geosparql/issues/316
    * @andrea-perego - I've essentially adopted your recommendations
    * Closes #316
* BFO: https://github.com/opengeospatial/ogc-geosparql/issues/355
    * @alanruttenberg - I think I've sorted out the inconsistency with relating Feature to "spatial region" and adopting your proposed Geometry mapping. Seems obvious in hindsight: "spatial region" should always have been mapped to Feature, not Geometry! Ooops me.
    * Closes #355

### CODE REVIEW

File: 1.1/spec/20-Annex-E.adoc [hunk 1]
```diff
@@ -237,7 +237,8 @@ DCTERMS Source: https://www.dublincore.org/specifications/dublin-core/dcmi-terms
...
+* http://purl.org/dc/terms/spatial[`dcterms:spatial`]: "Spatial characteristics of the resource". The range of this property includes a http://purl.org/dc/terms/Location[`dcterms:Location`], so it is a property for indicating a <<Class: geo:Feature, `geo:Feature`>>, for which GeoSPARQL has no equivalent, but perhaps also for indicating a <<Class: geo:Geometry, `geo:Geometry`>>, thus the subProertyOf mapping above.
```
Comment (lvdbrink) [inline 1.1/spec/20-Annex-E.adoc:240]: typo 'subProertyOf'

File: 1.1/spec/20-Annex-E.adoc [hunk 2]
```diff
@@ -387,15 +388,17 @@ BFO Source: https://basic-formal-ontology.org/bfo-2020.html, and from there, an
...
 | http://purl.obolibrary.org/obo/BFO_0000216[`obo:BFO_0000216`] "spatially projects onto at some time" | http://www.w3.org/2000/01/rdf-schema#range[`rdfs:range`] | <<Class: geo:SpatialObject, `geo:SpatialObject`>> | The reasoning is the same as for "occurs in"
 |===
+
+* BFO distinguishes between _continuants_ & _occurrants_, which _spatial region_ & _spatiotemporal region_ are subclasses of, respectively. GeoSPARQL has no handling of temporality, so cannot yet map to any _continuants_
+** a future version of GeoSPARQL that handled spatio-temporal Features could perhaps claim that <<Class: geo:Feature, `geo:Feature`>> is a http://www.w3.org/2000/01/rdf-schema#subClassOf[`rdfs:subClassOf`] http://purl.obolibrary.org/obo/BFO_0000011[`obo:BFO_0000011`] "spatiotemporal region", however inconsistencies from this mapping will occur due to the furrent Feature/"spatial region" mapping above and this will need to be handled
```
Comment (lvdbrink) [inline 1.1/spec/20-Annex-E.adoc:404]: typo 'furrent' > current

**Review by lvdbrink — APPROVED** [review]
Spotted 2 typos, otherwise good to merge!

**Review by situx — APPROVED** [review]

File: 1.1/spec/20-Annex-E.adoc (hunk 1 above)
Comment (nicholascar) [reply 1.1/spec/20-Annex-E.adoc]: Thanks, fixed in https://github.com/opengeospatial/ogc-geosparql/pull/363/commits/289bda4e58da958f8487b926117ebd3bb1ced467

**Pull request merged**